includes/class-wc-calypso-bridge-woocommerce-admin-features.php

```diff
@@ -97,6 +98,53 @@ public function filter_woocommerce_body_classes( $classes ) {
 		return $classes;
 	}
 
+	/**
+	 * Add is-woocommerce-home body class when url changes between wc-admin pages.
+	 *
+	 * @since   1.9.9
+	 *
+	 * @return void
+	 */
+	public function filter_woocommerce_body_classes_js() {
+		?>
+		<script>
+			document.addEventListener( 'DOMContentLoaded', function() {
+
+				// Bail out earlier.
+				if ( ! document.body.classList.contains( 'woocommerce_page_wc-admin' ) ) {
```

Comment:
I believe this won't work as expected (?) 

It falls in the same pitfall as the one that engaged this PR. React will always change the page contents, not the initial body classes, as the user navigates through other react pages.

Reply:
As we discussed

> `document.addEventListener( 'DOMContentLoaded', function() {`

Removed as it's not needed

> if ( ! document.body.classList.contains( 'woocommerce_page_wc-admin' ) ) {

Works fine, and it doesn't fall into the same pitfall.

Here's a video demonstrating that it works as expected


https://user-images.githubusercontent.com/2484390/204244914-15ffeb51-c161-4931-82e4-401de74bbee0.mp4